tests/pipelines/test_pipelines.py

```diff
@@ -99,6 +100,11 @@ def _test_from_save_pretrained_dynamo(in_queue, out_queue, timeout):
         scheduler = DDPMScheduler(num_train_timesteps=10)
 
         ddpm = DDPMPipeline(model, scheduler)
+
+        # previous diffusers versions stripped compilation off
+        # compiled modules
+        assert is_compiled_module(ddpm.unet)
+
```

Comment:
@patrickvonplaten just added to existing pipeline dynamo test. Confirmed fails on main and passes on branch

Reply:
Sounds good to me!